Assets/BossRoom/Scripts/Shared/Net/ConnectionManagement/GameNetPortal.cs

```diff
@@ -169,7 +174,7 @@ public void StartHost(string ipaddress, int port)
                 default:
                     throw new Exception($"unhandled IpHost transport {chosenTransport.GetType()}");
             }
-
+            BossRoomSessionManager.Instance.AddHostData(new SessionPlayerData(NetManager.LocalClientId, PlayerName, m_AvatarRegistry.GetRandomAvatar().Guid.ToNetworkGuid(), 0, true));
```

Comment:
why do we need to add a random selection on start host? Can't we wait for the player to select once in the char select scene?

Reply:
I added his here mostly to remove the responsibility of adding a random class on late join from BossRoomState. It also avoids [this issue](https://jira.unity3d.com/browse/MTT-1464)